.github/workflows/test_vendored.yml

```diff
@@ -1,4 +1,4 @@
-name: Test typing
+name: Test vendored
```

Comment:
And this also as otherwise it has the same name as .github/workflows/test_typing.yml